src/hooks/useBadgeCount.ts

```diff
@@ -0,0 +1,15 @@
+import { size } from "lodash";
+import { useInitialisedDeskproAppClient } from "@deskpro/app-sdk";
+import type { DeviceType } from "../types";
+
+const useBadgeCount = (items: DeviceType[]) => {
+  useInitialisedDeskproAppClient((client) => {
+    if (!Array.isArray(items)) {
```

Comment:
Items is always an array so this check isn't required.

Reply:
Fixed typing. I'm not sure it's always an array.

Reply:
I don't think that's the correct typing, `useBadgeCount` is always called with one argument, so it will always be an `array`.

Reply:
If you meant `(items?: DeviceType[]|undefined)` then I don't think that's a good signature and the function has unexpected behaviour.

If `undefined` is sent, why doesn't it set the count to `0`. Instead I strongly recommend we keep this simple and it always passes in an array and we just always call `client.setBadgeCount(...)`

Reply:
I change the function like this. The `items` come from the API, so I'm not sure if the array will come

<img width="399" alt="Screenshot 2024-07-18 at 13 17 02" src="https://github.com/user-attachments/assets/54cad280-18dc-4d2a-ae69-0d738eb5481f">